cves/kernel/CVE-2016-4578.yml

```diff
@@ -456,7 +479,11 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
```

Comment:
Replace "cogniscent" with "cognizant". You could replace the word "one" with "another" for clarity. Add punctuation.